includes/notes/class-wc-payments-notes-qualitative-feedback.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Set up refund policy note for WooCommerce inbox.
+ *
+ * @package WooCommerce\Payments\Admin
+ */
+
+use Automattic\WooCommerce\Admin\Notes\NoteTraits;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class WC_Payments_Notes_Set_Up_Refund_Policy
+ */
+class WC_Payments_Notes_Qualitative_Feedback {
+	use NoteTraits;
+
+	/**
+	 * Name of the note for use in the database.
+	 */
+	const NOTE_NAME = 'wc-payments-notes-qualitative-feedback';
+
+	/**
+	 * Name of the note for use in the database.
+	 */
+	const NOTE_DOCUMENTATION_URL = 'http://automattic.survey.fm/wc-pay-existing';
+
+	/**
+	 * Get the note.
+	 */
+	public static function get_note() {
+		global $wpdb;
+
+		// Store must be at least 60 days old.
+		if ( ! self::wc_admin_active_for( 60 * DAY_IN_SECONDS ) ) {
+			return;
+		}
+
+		// We should have at least one transaction.
+		$token_count = $wpdb->get_var( "select count(*) from {$wpdb->prefix}woocommerce_payment_tokens" );
+		if ( 0 === $token_count ) {
```

Comment:
This is a bit nitpicky, but I would prefer the logic `0 < $token_count` instead since the logic is more suitable to describe "at least one". Not a hard requirement though.

Reply:
Good suggestion. I will make the change.

Reply:
I agree with that 👍